### ISSUE TITLE
Typo on https://openshift.io/osiogettingstarted.pdf

### ISSUE THREAD

**Issue by nimishamukherjee** [openshiftio/openshift.io#1548]
https://openshift.io/osiogettingstarted.pdf
Page 25 - Change Approvel to Approval

**Comment by sbose78** [comment]
@qodfathr marking this as Sev2 because it doesn't look good to have typos in docs :)

**Comment by qodfathr** [comment]
Making a SEV4 because this doc will soon be obsolete... :)

**Comment by bartoszmajsak** [comment]
@rkratky @joshuawilson how this doc is ultimately generated and where is the repo we could send a PR to? Shall we look in https://github.com/fabric8io/fabric8-online-docs?

**Comment by sbose78** [comment]
@bartoszmajsak  https://github.com/openshiftio/openshift.io/tree/master/src/assets/documents

**Comment by rkratky** [comment]
@bartoszmajsak This PDF is deprecated (and has been for a long time). The URL should redirect either to https://docs.openshift.io/getting-started-guide.pdf or even better just to https://docs.openshift.io/

@sbose78 Could you make that redirect happen?

**Comment by bartoszmajsak** [comment]
Thanks, @sbose78 @rkratky, but those links don't really answer my question. In the spirit of open source I was hoping we can start doing is that if we see such a typo we simply open PR with the correction (ideally that should be caught during the build process of the PDF itself). So let me rephrase my question:

can you point me to the repository and build scripts using which this PDF is generated?

**Comment by bartoszmajsak** [comment]
On top of that. I don't see this PDF being available in the help section. 

![image](https://user-images.githubusercontent.com/719616/44023215-1e01c908-9eea-11e8-81e5-32ef15289192.png)

Documentation link points to `https://docs.openshift.io/`

Can we just delete this file?

**Comment by rkratky** [comment]
@bartoszmajsak What I'm saying is that the PDF (https://openshift.io/osiogettingstarted.pdf) should not be made available at all. It should be either deleted or redirected to the correct docs. There's no point in fixing it. It is not, and will not be, maintained.

Instead, all users should be pointed to the docs at docs.openshift.io -- which is where you'll also find a new, maintained PDF with the Getting Started Guide (https://docs.openshift.io/getting-started-guide.pdf), should you wish to use it.

If you find a mistake in the 'new', maintained docs, please, either submit an issue or a PR at https://github.com/fabric8io/fabric8-online-docs -- and the docs team will do its best to quickly fix it.

**Comment by rkratky** [comment]
> ideally that should be caught during the build process of the PDF itself

At this point, our CI doesn't check for typos, but we do implement a bunch of other checks to make reasonably sure the docs don't go out with mistakes. We'll look into checking for typos.

(Again, this is for the 'new' docs. The old PDF referenced by the OP was created by Todd who manually exported it from MS Word. Fortunately, we're over that :)

**Comment by bartoszmajsak** [comment]
> There's no point in fixing it. It is not, and will not be, maintained.

@rkratky so we are on the same page. We should make redirect perhaps, but I am just wondering if that link is by any means relevant - shared / exposed somehow. If not deleting is enough.

**Comment by rkratky** [comment]
> if that link is by any means relevant - shared / exposed somehow

It was promoted quite heavily (in registration e-mails) during the early days of OSIO, so I believe a redirect to docs.openshift.io would be better.

> maybe some GitHub ribbon or some other ways to encourage/promote collaboration in the docs could help to get those PRs?

Good idea. I filed https://github.com/fabric8io/fabric8-online-docs/issues/409 to track it.

**Comment by bartoszmajsak** [comment]
Wasn't aware of that heavy promotion, so yeah, a redirect will do. 

Closing in favor of #4162
